downstream/modules/automation-hub/proc-using-content-signing-services-in-pah.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc-using-content-signing-services-in-pah"]
+
+= Using content signing services in private automation hub
+
+After you have configured content signing on your {PrivateHubName}, you can manually sign a new collection or replace an existing signature with a new one so that users who want to download a specific collection have the assurance that the collection is intended for them and has not been modified after certification.
+
+Content signing on {PrivateHubName} provides solutions for the following scenarios:
+
+* Your system does not have automatic signing configured and you must use a manual signing process to sign collections.
+* The current signatures on the automatically configured collections are corrupted and must be replaced with new signatures.
+* Additional signatures are required for previously signed content.
+* You want to rotate signatures on your collections.
+
+.Procedure
+
+. In the `automationhub-terminal`, run the following command to publish your collections:
++
+----
+ansible-galaxy collection publish <ansible-test_collection-1.0.0.tar.gz> -c
+----
++
+[NOTE]
+====
+The `-c` option is used to ignore SSL certificate errors.
+====
+. Log in to your {PrivateHubName} instance in the {HubName} UI.
+
+. Use an unnumbered bullet (*) if the procedure includes only one step.
+
+. In the left navigation, click *Collections > Approval*.
+The Approval dashboard is displayed with a list of collections.
+
+. Click the *Sign and approve* button for each collection you want to prefer to sign.
```

Comment:
We absolutely want to show customers how its looks when you have configured signing, like screenshots of Approval page and how a collection looks when signed, but the constructs used were from self-paced lab and limited to it.
We just need to keep it generic.

Reply:
@anshulbehl - we are usually discouraged from using screenshots unless it's used in the conceptual topics where it enhances the communication. We are encouraged to use words to help the user understand what they are seeing. One of the reasons we are discouraged from using too many screenshots is due to changes that might occur in the UI. The concern is that it would be more taxing keeping up with the changes since UIs might change. I haven't heard that we translate our documentation, but I'm sure that is another reason for minimizing the number of screen shots. I can check with my team, but I'm pretty certain managing the updates to screenshots is a primary reason for not including them in task topics.

Reply:
Sounds good @jself-sudoku